tests/acceptance/features/webUISharingExternal/federationSharing.feature

```diff
@@ -47,209 +49,79 @@ Feature: Federation Sharing - sharing with users on other cloud storages
-    Then user "user2" should see the following elements
-      | /lorem%20(2).txt |
-
-  @skip @yetToImplement
-  Scenario: User-based accepting from trusted server checkbox is not visible while global is disabled
-    Given parameter "autoAddServers" of app "federation" has been set to "1"
-    And user "user1" from server "REMOTE" has shared "simple-folder" with user "user1" from server "LOCAL"
-    And user "user1" from server "LOCAL" has accepted the last pending share
-    And the user has reloaded the current page of the webUI
-    And parameter "auto_accept_trusted" of app "federatedfilesharing" has been set to "no"
-    And parameter "autoAddServers" of app "federation" has been set to "0"
-    And the user has browsed to the personal sharing settings page
-    Then User-based auto accepting from trusted servers checkbox should not be displayed on the personal sharing settings page in the webUI
-
-  @skip @yetToImplement
-  Scenario: User-based & global auto accepting is enabled but remote server is not trusted
-    Given parameter "auto_accept_trusted" of app "federatedfilesharing" has been set to "yes"
-    And parameter "autoAddServers" of app "federation" has been set to "0"
-    And the user has browsed to the personal sharing settings page
-    When the user disables automatically accepting remote shares from trusted servers
-    And the user enables automatically accepting remote shares from trusted servers
-    And user "user1" from server "REMOTE" shares "/lorem.txt" with user "user1" from server "LOCAL" using the sharing API
-    Then user "user1" should not see the following elements
-      | /lorem%20(2).txt |
-
-  @skip @yetToImplement
-  Scenario: share a folder with an remote user and prohibit deleting - local server shares - remote server receives
-    When the user shares folder "simple-folder" with remote user "user1@%remote_server_without_scheme%" using the webUI
-    And the user sets the sharing permissions of "user1@%remote_server_without_scheme% (federated)" for "simple-folder" using the webUI to
-      | delete | no |
-    And user "user1" re-logs in to "%remote_server%" using the webUI
-    And the user accepts the offered remote shares using the webUI
-    And the user opens folder "simple-folder (2)" using the webUI
-    Then it should not be possible to delete file "lorem.txt" using the webUI
+    Given the setting "auto_accept_trusted" of app "federatedfilesharing" has been set to "yes"
+    And user "user1" from remote server has shared "simple-folder" with user "user1" from local server
+    When the user reloads the current page of the webUI
+    Then folder "simple-folder" should be listed on the webUI
+    When the user opens folder "simple-folder" directly on the webUI
+    Then file "lorem.txt" should be listed on the webUI
+    When the user browses to the shared-with-me page
+    Then folder "simple-folder" should not be listed on the webUI
```

Comment:
should be tagged with an issue, because that share should be listed there

Reply:
Done by 504d4a1